tasks/main.yml

```diff
@@ -6,6 +6,19 @@
   become: yes
   file: path={{coreos_kubelet_install_dir}} state=directory mode=0755
 
+- name: Create manifest dir
```

Comment:
Not sure why we need to create `/etc/kubernetes` directory (kubelet-wrapper already does that). Anyway we are not creating the manifest directory here (which is actually `/etc/kubernetes/manifests`).

Reply:
Yes, was using interlinked handler to restart kubelet. Directly using kubelet restart  handler now.

Reply:
Earlier I was copying kubelet-config at `/etc/kubernetes` before kubelet-wrapper gets called and the dir wasn't created at that time. Moved kubelet-config to a different dir later but the merge was wrong and it left that additional task. Cleaned up now.